test/test.js

```diff
@@ -761,6 +761,52 @@ test( "elementValue() finds radios/checkboxes only within the current form", fun
 	ok( !v.elementValue( foreignRadio ) );
 } );
 
+test( "elementValue() returns the file input's name without the prefix 'C:\\fakepath\\' ", function() {
+	var v = $( "#userForm" ).validate(),
+		fileInput = {
+			name: "fakeFile",
+			type: "file",
+			files: {},
+			nodeName: "INPUT",
+			value: "C:\\fakepath\\somefile.txt",
+			form: $( "#userForm" )[ 0 ],
+			hasAttribute: function() { return false; }
+		},
+		oldRules = $.fn.rules,
+		oldvalidationTargetFor = v.validationTargetFor;
+
+	equal( v.elementValue( fileInput ), "somefile.txt" );
+
+	$.fn.rules = function() {
+		return {
+			minlength: 10
+		};
+	};
+
+	v.validationTargetFor = function() {
+		return fileInput;
+	};
+
+	ok( v.check( fileInput ), "The fake file input is valid (length = 12, minlength = 10)" );
```

Comment:
Because I get this error, and it's caused because of `defaultShowErrors()`:
```
at http://127.0.0.1:34545/test/test.js:764:1: Cannot read property 'createDocumentFragment' of undefined@ 8 ms
Source: 	
TypeError: Cannot read property 'createDocumentFragment' of undefined
    at createSafeFragment (http://127.0.0.1:34545/lib/jquery-1.11.1.js:5274:22)
    at Function.jQuery.extend.buildFragment (http://127.0.0.1:34545/lib/jquery-1.11.1.js:5545:11)
    at jQuery.fn.extend.domManip (http://127.0.0.1:34545/lib/jquery-1.11.1.js:5910:22)
    at jQuery.fn.extend.after (http://127.0.0.1:34545/lib/jquery-1.11.1.js:5755:15)
    at jQuery.each.jQuery.fn.(anonymous function) [as insertAfter] (http://127.0.0.1:34545/lib/jquery-1.11.1.js:5987:35)
    at Object.$.extend.showLabel (http://127.0.0.1:34545/dist/jquery.validate.js:893:12)
    at Object.$.extend.defaultShowErrors (http://127.0.0.1:34545/dist/jquery.validate.js:829:10)
    at Object.$.extend.showErrors (http://127.0.0.1:34545/dist/jquery.validate.js:492:10)
    at Object.$.extend.element (http://127.0.0.1:34545/dist/jquery.validate.js:466:9)
    at Object.<anonymous> (http://127.0.0.1:34545/test/test.js:798:9)
```